### DISCUSSION TITLE
Unable to add RSS from vwidtalk.com

### DISCUSSION THREAD

**Discussion by astrodad** [FreshRSS/FreshRSS#5510] [Q&A] [answered]
**Describe the bug**
I am getting a 403 error in the logs when attempting to add this RSS feed, which appears valid and works elsewhere: https://www.vwidtalk.com/forums/-/index.rss I was able to run a manual curl on this URL on the server itself and it also works fine.

**To Reproduce**
1. Try to add a new subscription to https://www.vwidtalk.com/forums/-/index.rss
2. See error

**Expected behavior**
The feed should be added. 

**Environment information (please complete the following information):**
- Device: Chrome
- OS: macOS
- Browser: [e.g. Firefox 86]
- FreshRSS version: 1.21.0
- Installation type:  self-hosted

**Comment by Alkarex** [comment, marked as answer]
Hello,
It looks like the server is forbidding requests, using different rules including at least user-agent.
Try to use `curl` in the user-agent field when adding the feed. It worked for me:

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/bf7d2221-c2bb-4126-8e33-f44330a69a53)

**Comment by astrodad** [comment]
Perfect, thanks!!
